source/administration/config-settings.rst

```diff
@@ -1126,6 +1126,16 @@ Enable Email Notifications
 | This feature's ``config.json`` setting is ``"SendEmailNotifications": false`` with options ``true`` and ``false`` for above settings respectively.                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Email Batching
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**True**: Users can have email notifications for multiple direct messages and mentions combined into a single email, configurable in **Account Settings** > **Notifications**. Note: Email batching cannot be enabled unless the `SiteURL <https://docs.mattermost.com/administration/config-settings.html#site-url>`_ is configured and `High Availability <https://docs.mattermost.com/administration/config-settings.html#enable-high-availability-mode>`_ is disabled.
```

Comment:
Can we say "Users can select how often to receive email notifications, and multiple notifications within that timeframe will be combined into a single email."  or something like that? 

I think the "how often" part is the main benefit, so it would be nice to include that somehow